File: Level-1/43.java
import java.util.*;
class Solution {
    public int[] solution(int[] numbers) {
        int[] answer;
        ArrayList<Integer> list = new ArrayList<>();
        for(int i = 0; i < numbers.length; i++){
            for(int j = i + 1; j < numbers.length; j++){
                int sum = numbers[i] + numbers[j];
                if(!list.contains(sum)) list.add(sum);
            }
        }
        answer = new int[list.size()];
        for(int i = 0; i < list.size(); i++){
            answer[i] = list.get(i);
        }
        Arrays.sort(answer);
        return answer;
    }
}

File: Level-1/43.js
function solution(numbers) {
    const set = new Set();
    
    numbers.forEach((number, index) => {
        for(let i = 0; i < numbers.length; i++) {
            if(index !== i) {
                set.add(number + numbers[i]);
            }
        }
    });
    
    return [...set].sort((a, b) => a - b);
}
